notebooks/ImvizExample.ipynb

```diff
@@ -184,7 +184,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data = imviz.app.get_data_from_viewer('viewer-1', 'gc_2mass_j')"
+    "data = imviz.app.get_data_from_viewer('viewer-1', 'gc_2mass_j[PRIMARY,1]')"
```

Comment:
Another oversight from parser work that I am just going to fix here. But if this takes too long, I'll split this out into a different PR.